src/Pages/Auth/Components/InputAndLabel.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+
+const InputAndLabel = props => {
```

Comment:
```suggestion
const InputAndLabel = ({ format }) => {
```
- props 객체를 바로 구조 분해 할당 적용할 수 있습니다!

Reply:
감사합니다 바로 적용시켰습니다!!